packages/caver-klay/caver-klay-accounts/src/index.js

```diff
@@ -544,7 +590,8 @@ Accounts.prototype.decrypt = function (v3Keystore, password, nonStrict) {
       "id":"dfde6a32-4b0e-404f-8b9f-2b18f279fe21",
     }
  */
-Accounts.prototype.encrypt = function (privateKey, password, options) {
+// The key parameter can be either normal private key or klaytnWalletKey format.
```

Comment:
Can we also add a description for this function?

Reply:
I added description for encrypt function :)